Parser/pegen.c

```diff
@@ -525,10 +525,13 @@ _PyPegen_dummy_name(Parser *p, ...)
 static int
 _get_keyword_or_name_type(Parser *p, const char *name, int name_len)
 {
-    if (name_len >= p->n_keyword_lists || p->keywords[name_len] == NULL) {
+    assert(name_len != 0);
+    if (name_len >= p->n_keyword_lists ||
+        p->keywords[name_len] == NULL ||
```

Comment:
Just asking, but, theoretically, this should never be `NULL`, right?

Reply:
Indeed, but I decided to leave it in case we put `NULL` in the future, to avoid spooky action-at-a-distance :) 

(Reference to: https://www.sciencemag.org/news/2018/04/einstein-s-spooky-action-distance-spotted-objects-almost-big-enough-see :atom: )